Grammar/python.gram

```diff
@@ -780,7 +787,7 @@ primary[expr_ty]:
 
 slices[expr_ty]:
     | a=slice !',' { a }
-    | a[asdl_expr_seq*]=','.slice+ [','] { _PyAST_Tuple(a, Load, EXTRA) }
+    | a[asdl_expr_seq*]=','.(slice | starred_expression)+ [','] { _PyAST_Tuple(a, Load, EXTRA) }
```

Comment:
> I think what happens is, if we move `starred_expression` to `slice` (as `| a=starred_expression { a }`), then when we do `a[*b]`, it triggers the _first_ option of `slices`, where the `starred_expression` _doesn't_ get processed by `_PyAST_Tuple`.

I am not sure I follow, but don't worry, I will check what's going on here. I did the original suggestions over the phone so I didn't sit down and check that those were actually viable. 

I am still a bit confused, because:

```
a = c | b
c = c1 | c2
```
 should be equivalent to

```
a = c | c2 | b
c = c1
```

Reply:
Ok, fair enough :)